tests/test_intervals.py

```diff
@@ -11,6 +11,16 @@
 
 @pytest.fixture(scope="module")
 def test_data():
+    """
+    Function to create test data.
+
+    Returns
+    -------
+    Numpy array
+        Test data.
+
+    """
+    
     return np.array([1.2, 1.5, 2.4, 2.5, 2.6, 3.1, 3.5, 3.6, 4.0, 5.0])
```

Comment:
These numbers look the same as the ones used for "ref_intervals" (which I do not understand).

Reply:
These are test data which should be sorted into the intervals.